rustic-babel.el

```diff
@@ -342,9 +342,20 @@ executed with the parameter `:include'."
 (defun rustic-babel-block-contents (block-name)
   "Return contents of block with the name BLOCK-NAME"
   (with-current-buffer (current-buffer)
-    (save-excursion
-      (org-babel-goto-named-src-block block-name)
-      (org-element-property :value (org-element-at-point)))))
+    (cond
+     ;; Block exists in current file
+     ((org-babel-find-named-block block-name)
+      (save-excursion
+        (org-babel-goto-named-src-block block-name)
+        (org-element-property :value (org-element-at-point))))
+     ;; Block exists in library of babel
+     ;; (see https://orgmode.org/manual/Library-of-Babel.html)
+     ((nth 2 (assoc-string block-name org-babel-library-of-babel)))
+     ;; Fallback
+     (t
+      (progn
+        (message "included source code block `%s' not found in this buffer or the library of babel" block-name)
```

Comment:
so is the only thing this PR is doing is showing a message if the block doesn't exist?

Reply:
No, the main thrust is the attempt to load it from `org-babel-library-of-babel` above. The error message is just reintroducing the error that `goto` would normally produce (w/ a clarification about lib-of-babel also being supported).